flexport/commentariat
# Upgraded babel dependencies and flow

This PR upgrades commentariat's babel related dependencies and `.babelrc` to match formula-one's (https://github.com/flexport/formula-one/blob/master/.babelrc), and upgrades flow to the latest version.

To test these upgrades:
* make sure all tests are passing via `yarn test`
* make sure there are no flow violations by running `yarn flow`
